### ISSUE TITLE
Runtime Error: unexpected number of Arguments when running python -m colosseum.tools.visualize_task --config-name close_box

### ISSUE THREAD

**Issue by sadeghkt** [robot-colosseum/robot-colosseum#4]
Hello good people,

when I run the the `python -m colosseum.tools.visualize_task --config-name close_box` in terminal in my py environment after following all install steps, I get the following error:

``` bash
Error executing job with overrides: []
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/path/robot-colosseum/colosseum/tools/visualize_task.py", line 47, in <module>
    SystemExit(main())
               ^^^^^^
  File "/path/vcolloseum/lib/python3.12/site-packages/hydra/main.py", line 94, in decorated_main
    _run_hydra(
  File "/path/vcolloseum/lib/python3.12/site-packages/hydra/_internal/utils.py", line 394, in _run_hydra
    _run_app(
  File "/path/vcolloseum/lib/python3.12/site-packages/hydra/_internal/utils.py", line 457, in _run_app
    run_and_report(
  File "/path/vcolloseum/lib/python3.12/site-packages/hydra/_internal/utils.py", line 223, in run_and_report
    raise ex
  File "/path/vcolloseum/lib/python3.12/site-packages/hydra/_internal/utils.py", line 220, in run_and_report
    return func()
           ^^^^^^
  File "/path/vcolloseum/lib/python3.12/site-packages/hydra/_internal/utils.py", line 458, in <lambda>
    lambda: hydra.run(
            ^^^^^^^^^^
  File "/path/vcolloseum/lib/python3.12/site-packages/hydra/_internal/hydra.py", line 132, in run
    _ = ret.return_value
        ^^^^^^^^^^^^^^^^
  File "/path/vcolloseum/lib/python3.12/site-packages/hydra/core/utils.py", line 260, in return_value
    raise self._return_value
  File "/path/vcolloseum/lib/python3.12/site-packages/hydra/core/utils.py", line 186, in run_job
    ret.return_value = task_function(task_cfg)
                       ^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/robot-colosseum/colosseum/tools/visualize_task.py", line 25, in main
    env = EnvironmentExt(
          ^^^^^^^^^^^^^^^
  File "/path/robot-colosseum/colosseum/rlbench/extensions/environment.py", line 86, in __init__
    raise RuntimeError(
RuntimeError: Unexpected number of arguments in Environment.__init__
```

I followed all install steps from [here](https://robot-colosseum.readthedocs.io/en/latest/installation.html). I set everything up on WSL Ubuntu 22.04.

I appreciate any help and support.

**Comment by wpumacay** [comment]
Hi @sadeghkt . This issue is related to a mismatch of the version of RLBench that we used for this project, and the latest version of RLBench which most likely you're using. Sorry, in this script I just did a hack to check for the right `Environment` API and call accordingly, [here](https://github.com/robot-colosseum/robot-colosseum/blob/5ffbf53d2d1d49c0041e774d3109f64c36c30a09/colosseum/rlbench/extensions/environment.py#L53). This can be fixed by downgrading RLBench to a version close enough to the one we used, like this one stepjam/rlbench@7c3f425f4a0b6b5ce001ba7246354eb3c70555be .

Another option is to use the RLBench fork from `PerAct` [here](https://github.com/MohitShridhar/RLBench/tree/peract). May I ask, are you trying to reproduce the results from PerAct or RVT? if that's the case you would most likely need to use the PerAct fork of RLBench then, because it contains some changes to the `Demo` and `Observation` classes which they used to train their models. If you collect the data with this fork then you'll have these extra fields pickled into the saved data. Otherwise, when you try to use your data after generating it, your baseline might not work because of this mismatch in the pickled data. I'll update the docs and try to find a better solution for both these issues.

Let me know if this fixes the issue, or if you run into any other issues.
